### DISCUSSION TITLE
On-Demand Cache Invalidation

### DISCUSSION THREAD

**Discussion by siolfyr** [nuxt/nuxt#22311] [Questions]
Does this exist in Nuxt? Next does this by route or by cache tag:
https://nextjs.org/docs/app/building-your-application/data-fetching/revalidating

I'd like to use stale-while-revalidate but optionally manually revalidate the cache. Or at the very least completely blow it away so I can hit the route myself and regenerate it. But I'm not clear on this based on the docs.

**Comment by maximilliangeorge** [comment]
I don't think it exists at time of writing, but this would be an amazing feature.
